dask_expr/_collection.py

```diff
@@ -103,6 +103,62 @@ def _wrap_expr_op(self, other, op=None):
         )
 
 
+def _wrap_expr_method_operator(name, class_):
+    """
+    Add method operators to Series or DataFrame like DataFrame.add.
+    _wrap_expr_method_operator("add", DataFrame)
+    """
+    if class_ == DataFrame:
+
+        def method(self, other, axis="columns", level=None, fill_value=None):
+            if level is not None:
+                raise NotImplementedError("level must be None")
+            if not isinstance(other, DataFrame):
```

Comment:
Why is this not implemented?

Reply:
Oh no that's fine, can you add a todo though?

Reply:
https://github.com/dask-contrib/dask-expr/pull/522/commits/01b1517405883124dac7848cb495beb986d42eb9 done, will follow up tomorrow.